korebuild.json

```diff
@@ -1,11 +1,11 @@
 {
   "$schema": "https://raw.githubusercontent.com/aspnet/BuildTools/dev/tools/korebuild.schema.json",
-  "channel": "dev",
+  "channel": "release/2.1",
   "toolsets": {
     "visualstudio": {
       "required": false,
       "includePrerelease": true,
-      "minVersion": "15.0.26730.03",
+      "minVersion": "[15.0.26730.03, 15.8)",
```

Comment:
We need to put an upper bound on which versions of MSBuild we use to build the VSIX. The current preview of VS isn't compatible with this SDK for building **BUILDING THE VSIX AT THE COMMANDLINE ONLY**.

I'm going to check on this before merging this PR and see if there's another option. This is the setup that razor is using currently.